logstash/config/logstash.conf

```diff
@@ -0,0 +1,13 @@
+input {
+	udp {
```

Comment:
[memo]
I was wondering if I would use gelf, but for some reason gelf doesn't collect logs. I'm not sure.

```sh
gelf {}
```

Logs can now be collected using udp, but parsing and codecs need to be adjusted.